cluster/gce/windows/node-helper.sh

```diff
@@ -52,7 +52,9 @@ function get-windows-node-instance-metadata {
 # $1: template name (required).
 # $2: scopes flag.
 function create-windows-node-instance-template {
-  local template_name="$1"
-  local scopes_flag="$2"
-  create-node-template "${template_name}" "${scopes_flag}" "$(get-windows-node-instance-metadata-from-file)" "$(get-windows-node-instance-metadata)" "windows"
+  if [[ "${NUM_WINDOWS_NODES}" -gt "0" ]]; then
+    local template_name="$1"
+    local scopes_flag="$2"
+    create-node-template "${template_name}" "${scopes_flag}" "$(get-windows-node-instance-metadata-from-file)" "$(get-windows-node-instance-metadata)" "windows"
```

Comment:
What's the point of this change? The fact that we don't create windows nodes at the beginning doesn't mean we won't want to do that later...

Reply:
As stated in the PR description: I want to disable creation of Windows template. In scalability tests we don't use them at the moment and it takes some time to create it.

Reply:
Saving 3s is not worth it imho.